packages/knip/src/plugins/changelogen/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { toC12config } from '../../util/plugin-config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://github.com/unjs/changelogen
+
+const title = 'Changelogen';
+
+const enablers = ['changelogen', 'changelogithub'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const resolveConfig = () => {
+  return [];
+};
+
+const config = ['package.json', ...toC12config('changelog')];
```

Comment:
We should probably use `entry` instead if we don't actually need the configuration object. And in that case omit `package.json`.